components/back-to-top/src/index.scss

```diff
@@ -0,0 +1,45 @@
+/* Back to top button */
+
+.back-to-top {
```

Comment:
All of these elements are inside the custom element right? If so let's make these more specific by wrapping them in a custom element specifier. If styles aren't inside a custom element use a cagov- prefix on the classnames

Reply:
Not this one... this is appended as #page-container child

Reply:
OK 

Want to use:

```
.cagov-back-to-top {
```

We've been starting to follow that pattern in other components so there is less risk of overrides when used on someone else's site.

Reply:
I have put it inside of the custom element and wrapped css too.